pkgs/by-name/ni/ninja/package.nix

```
@@ -11,18 +11,35 @@
 , buildPackages
 , buildDocs ? true
 , nix-update-script
+, ninjaRelease ? "latest"
```

```suggestion
, use_1_11 ? false
```

nit. a bool rather than a string would eliminate the error case of an invalid string and make the below assert unnecessary